From b414047c540d8a801a5951457af5ab9b6f217cbe Mon Sep 17 00:00:00 2001
From: lihuihui <lihuihui@eswincomputing.com>
Date: Thu, 3 Jul 2025 18:57:19 +0800
Subject: [PATCH 611/652] feat:support eswin's hwrng

Changelogs:
1.support eswin's hwrng by CONFIG_CRYPTO_DEV_ESWIN_EIC770x=y

Signed-off-by: lihuihui <lihuihui@eswincomputing.com>
Reviewed-by: Joey Zheng <zhengyu@eswincomputing.com>
---
 arch/riscv/configs/vpu7702_evb_defconfig     | 1 +
 arch/riscv/configs/vpu7702_pcie_a2_defconfig | 1 +
 arch/riscv/configs/vpu7702_pcie_defconfig    | 1 +
 3 files changed, 3 insertions(+)

diff --git a/arch/riscv/configs/vpu7702_evb_defconfig b/arch/riscv/configs/vpu7702_evb_defconfig
index 36756a8dfd50..59589ede0dc1 100644
--- a/arch/riscv/configs/vpu7702_evb_defconfig
+++ b/arch/riscv/configs/vpu7702_evb_defconfig
@@ -829,6 +829,7 @@ CONFIG_CRYPTO_MD5=y
 CONFIG_CRYPTO_SHA512=m
 CONFIG_CRYPTO_AES=m
 CONFIG_CRYPTO_DEV_VIRTIO=y
+CONFIG_CRYPTO_DEV_ESWIN_EIC770x=y
 CONFIG_CRC_ITU_T=y
 CONFIG_CRC7=y
 CONFIG_DMA_CMA=y
diff --git a/arch/riscv/configs/vpu7702_pcie_a2_defconfig b/arch/riscv/configs/vpu7702_pcie_a2_defconfig
index e70e59a4fb41..2a1cbf1ba2bf 100644
--- a/arch/riscv/configs/vpu7702_pcie_a2_defconfig
+++ b/arch/riscv/configs/vpu7702_pcie_a2_defconfig
@@ -829,6 +829,7 @@ CONFIG_CRYPTO_MD5=y
 CONFIG_CRYPTO_SHA512=m
 CONFIG_CRYPTO_AES=m
 CONFIG_CRYPTO_DEV_VIRTIO=y
+CONFIG_CRYPTO_DEV_ESWIN_EIC770x=y
 CONFIG_CRC_ITU_T=y
 CONFIG_CRC7=y
 CONFIG_DMA_CMA=y
diff --git a/arch/riscv/configs/vpu7702_pcie_defconfig b/arch/riscv/configs/vpu7702_pcie_defconfig
index 45b98b871a63..6c20799e4b52 100644
--- a/arch/riscv/configs/vpu7702_pcie_defconfig
+++ b/arch/riscv/configs/vpu7702_pcie_defconfig
@@ -829,6 +829,7 @@ CONFIG_CRYPTO_MD5=y
 CONFIG_CRYPTO_SHA512=m
 CONFIG_CRYPTO_AES=m
 CONFIG_CRYPTO_DEV_VIRTIO=y
+CONFIG_CRYPTO_DEV_ESWIN_EIC770x=y
 CONFIG_CRC_ITU_T=y
 CONFIG_CRC7=y
 CONFIG_DMA_CMA=y
-- 
2.51.1

